### PR TITLE
docs(migration): add code migration guide

### PR DESCRIPTION
**Pull request by lucasaarcoverde**
## Summary

Add the admin-ui to shoreline code migration guide

### CODE REVIEW

File: packages/docs/pages/guides/migration/code.mdx [hunk 1]
```diff
@@ -0,0 +1,530 @@
+# Code Migration
+
+In this documentation you can find information about what changed from `@vtex/admin-ui@0latest` to `@vtex/shoreline` package.
+
+## Alert
+
+* The `action` property doesn’t exist anymore. Now you must add the action by hand using the `Button` component
+* the variant property values has changed from `'info' | 'positive' | 'warning' | 'critical'` to ` 'informational' | 'success' | 'critical' | 'warning'`
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:8]:
```suggestion
* The `variant` property values have changed from `'info' | 'positive' | 'warning' | 'critical'` to ` 'informational' | 'success' | 'critical' | 'warning'`.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+  <Text variant="body">Action as button</Text>
+  <Button variant="tertiary" onClick={() => alert('Clicked')}>
+    Action
+  </Button>
+</Alert>
+```
+
+## Bleed
+
+* the properties `left` and `right` are now `start` and `end`.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:38]:
```suggestion
* The properties `left` and `right` are now `start` and `end`.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+```jsx
+<Bleed start="" end="" bottom="" top="">
+  ...
+</Bleed>
+```
+
+
+## Button
+
+* the `variant` property values has changed from `primary | secondary | tertiary | critical |criticalSecondary |criticalTertiary | neutralTertiary` to `'primary' | 'secondary' | 'tertiary' | 'critical' | 'criticalTertiary'`
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:57]:
```suggestion
* The `variant` property values have changed from `primary | secondary | tertiary | critical |criticalSecondary |criticalTertiary | neutralTertiary` to `'primary' | 'secondary' | 'tertiary' | 'critical' | 'criticalTertiary'`
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+  <FieldError />
+</Field>
+```
+
+
+## DataView
+
+This component was replaced by the `Collection` component.
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:106]:
```suggestion
There are several changes in the component API. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+    helpText=""
+  />
+  <FieldDescription />
+  <FieldError />
+</Field>
+```
+
+## Filter
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:155]:
```suggestion
There are several changes in the component API. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+There are no changes.
+
+## Grid
+
+There are no changes.
+
+## Heading
+
+We've removed this component, now you must use specific components such as `ModalHeading` and `PageHeading` on their context.
+You can check their documentation for detailed information.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:199]:
```suggestion
Check their documentation for detailed information.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 7]
```diff
@@ -0,0 +1,530 @@
...
+
+* We’ve added the `optional` property that indicates whether the form field is optional or not.
+
+## Anchor
+
+* The name has changed from `Anchor` to `Link`
+
+## Menu
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:216]:
```suggestion
There are several changes in the component API. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 8]
```diff
@@ -0,0 +1,530 @@
...
+```
+
+## Modal
+
+We split the Modal component into two components:
+
+`ConfirmationModal`: allow merchants to confirm an action through an overlay window that opens on top of the current page.
+`Modal`: allow users to view content that demands attention through an overlay window that opens on top of the current page.
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:262]:
```suggestion
There are several changes in the component AP. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 9]
```diff
@@ -0,0 +1,530 @@
...
+
+## Page
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:266]:
```suggestion
There are several changes in the component AP. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 10]
```diff
@@ -0,0 +1,530 @@
...
+```
+
+**After**
+```jsx
+<Stack horizontal>...</Stack>
+```
+
+## Tab
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:386]:
```suggestion
There are several changes in the component AP. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 11]
```diff
@@ -0,0 +1,530 @@
...
+    <Tab>Tab 2</Tab>
+  </TabList>
+  <TabPanel />
+  <TabPanel />
+</TabProvider>
+```
+
+## Table
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:416]:
```suggestion
There are several changes in the component AP. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 12]
```diff
@@ -0,0 +1,530 @@
...
+  <TooltipPopover>
+    <TooltipArrow />
+    Tooltip text
+  </TooltipPopover>
+</TooltipProvider>
+```
+
+## Toast
+
+There are several changes in the component API, we’d recommend you to check the documentation before doing the migration.
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:478]:
```suggestion
There are several changes in the component AP. Check the documentation before migrating.
```

File: packages/docs/pages/guides/migration/code.mdx [hunk 13]
```diff
@@ -0,0 +1,530 @@
+# Code Migration
+
+In this documentation you can find information about what changed from `@vtex/admin-ui@0latest` to `@vtex/shoreline` package.
+
+## Alert
+
+* The `action` property doesn’t exist anymore. Now you must add the action by hand using the `Button` component
```
Comment (beatrizmilhomem) [inline packages/docs/pages/guides/migration/code.mdx:7]:
```suggestion
* The `action` property doesn’t exist anymore. Now you must add the action by hand using the `Button` component.
```

**Comment by davicostalf** [comment]
@lucasaarcoverde @beatrizmilhomem What do you think about changing the section title from just "Migration" to "Admin UI Migration"?

**Pull request merged**